src/averageAges.js

```diff
@@ -14,14 +14,23 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+const MALE = 'f';
+const FEMALE = 'f';
+const ONE_CENTURY = 1000;
+
+function menAverageAge(people, century) {
```

Comment:
The function name `menAverageAge` does not follow the naming convention. It should start with a verb to clearly indicate its purpose, such as `calculateMenAverageAge`.